_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2077,6 +2077,14 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'jp_group'          => 'search',
 			),
 
+			'has_jetpack_search_product'           => array(
+				'description'       => esc_html__( 'Has an active Jetpack Search product purchase', 'jetpack' ),
+				'type'              => 'boolean',
+				'default'           => 0,
+				'validate_callback' => __CLASS__ . '::validate_boolean',
+				'jp_group'          => 'search',
```

Comment:
Would deleting this `jp_group` value prevent module de/activations from mutating this option value?

Reply:
I think I added this because someone suggested it at some point. I was thinking it had more to do with documentation and grouping of the options in the api, but I am unsure.